ngSe/exceptions.py

```diff
@@ -47,6 +47,7 @@ class FrontEndError(Exception):
     selenium_exceptions.InvalidElementStateException,
     selenium_exceptions.NoSuchElementException,
     selenium_exceptions.ElementNotVisibleException,
+    selenium_exceptions.StaleElementReferenceException,
```

Comment:
Had it ran against the previously failing code, no more stale element exceptions